mws/apis/offamazonpayments.py

```diff
@@ -17,63 +21,52 @@ class OffAmazonPayments(MWS):
 
     def authorize(self, order_ref, order_total, auth_id, timeout=60):
         return self.make_request(
-            extra_data=dict(
-                Action="Authorize",
-                AmazonOrderReferenceId=order_ref,
-                AuthorizationReferenceId=auth_id,
-                TransactionTimeout=timeout,
-                **{
-                    "AuthorizationAmount.Amount": "{:.2f}".format(order_total),
-                    "AuthorizationAmount.CurrencyCode": "USD",
-                }
-            )
+            "Authorize",
```

Comment:
Without following this through so much to completion to check (sorry) why has the signature changed from `extra_data` to just all being in the main data object?

Reply:
ignore these, the argument was called extra_data in make_request, but we have been making data objects elsewhere. 

new way is cleaner